src/seat.c

```diff
@@ -95,8 +95,26 @@ configure_libinput(struct wlr_input_device *wlr_input_device)
 			dc->tap_button_map);
 	}
 
+	if (!libinput_device_config_tap_get_enabled(libinput_dev)
```

Comment:
Consider adding a finger-count check.

I had a look at the sway code. They only apply `tap_drag` and `drag_lock` if `finger-count > 0` and 

https://github.com/swaywm/sway/blob/89f85312687b7fd1777c936e169e6400cee0a4b4/sway/input/libinput.c#L50-L70

Reply:
That makes sence.

In my previous code, configuring `tap-and-drag` required `tap` to be enabled.
And [`libinput_device_config_tap_get_enabled` checked finger count internally](https://gitlab.freedesktop.org/libinput/libinput/-/blob/99c4721f95ccf50bc8153b53938cb1f3160ad450/src/libinput.c#L3894).
Same goes for `drag lock`, which required `tap-and-drag` to be enabled.

But sway's approach looks simpler and it doesn't make any practical difference.